src/main/java/nextstep/subway/line/LineService.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.subway.line;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import nextstep.subway.common.exception.ResourceNotFoundException;
+import nextstep.subway.section.Section;
+import nextstep.subway.station.Station;
+import nextstep.subway.station.StationRepository;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
+
+    public LineService(final LineRepository lineRepository,
+        final StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public LineResponse saveLine(final LineRequest lineRequest) {
+        Line line = new Line(lineRequest.getName(),
+            lineRequest.getColor(),
+            new Section(findStationById(lineRequest.getUpStationId()),
+                findStationById(lineRequest.getDownStationId()),
+                lineRequest.getDistance())
+        );
```

Comment:
저는 오히려 명확하게 어떤 값들을 사용하는지 표시해 주는 쪽이 가독성이 더 좋다고 생각해요. 
```suggestion
        Station upStation = findStationById(lineRequest.getUpStationId());
        Station downStation = findStationById(lineRequest.getDownStationId());
        Section section = new Section(upStation, downStation, lineRequest.getDistance());
        
        Line line = new Line(lineRequest.getName(), lineRequest.getColor(), section);
```